pkg/flow/grpc-instances.go

```diff
@@ -30,15 +30,17 @@ func (flow *flow) InstanceInput(ctx context.Context, req *grpc.InstanceInputRequ
 	if err != nil {
 		return nil, err
 	}
-
+	err = atob(d.in.Edges.Orginator, &resp.Instance.Originator)
```

Comment:
`atob` is my least helpfully named function, which is especially bad because it's quite complex. I think if we design the gRPC api correctly you shouldn't ever need to do this.